samples/sample_common/src/vaapi_utils_drm.cpp

```diff
@@ -82,13 +82,34 @@ int open_intel_adapter(int type)
     return -1;
 }
 
-DRMLibVA::DRMLibVA(int type)
+int open_intel_adapter(int type, const char * devicePath)
+{
+    char driverName[MFX_DRM_DRIVER_NAME_LEN + 1] = {};
+
+    if(strcmp(devicePath,"") == 0)
+        return open_first_intel_adapter(type);
+
+    int fd = open(devicePath, O_RDWR);
+
+    if (fd < 0) {
+        close(fd);
+    }
+
+    if (!get_drm_driver_name(fd, driverName, MFX_DRM_DRIVER_NAME_LEN) &&
+        !strcmp(driverName, MFX_DRM_INTEL_DRIVER_NAME)) {
+            return fd;
```

Comment:
add printf and describe that specified adapter is not intel one?

Reply:
Done